mars/services/task/execution/ray/executor.py

```diff
@@ -149,6 +150,7 @@ def execute_subtask(
     -------
         subtask outputs and meta for outputs if `output_meta_keys` is provided.
     """
+    init_metrics("ray")
```

Comment:
Could pass the metric backend parameter rather than a fixed value? We may change the backend on some scenario like change it to `console` if do not record metrics.

Reply:
> Could pass the metric backend parameter rather than a fixed value? We may change the backend on some scenario like change it to `console` if do not record metrics.

Then, we should pass the backend param every subtask. Any better solution?

Reply:
I have removed the init metrics at execute_subtask. This fix is not related with this PR. We can fix the metrics in the future.